src/pocketmine/resources/pocketmine.yml

```diff
@@ -1,84 +1,148 @@
+# Main configuration file for ClearSky
+# Some of these settings are safe, others can break your server if modified incorrectly!
+# If config-update is set to true, new settings/defaults will appear automatically on this file when upgrading,
+
 settings:
+ #Three-letter language code for server-side localization
+ #Check your language code on https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes
  language: "eng"
  force-language: false
+ #Whether to send all strings translated to server locale or let the device handle them
  shutdown-message: "Server closed"
  whitelist-message: "Server is whitelisted"
  query-plugins: true
+ #Allow listing plugins via Query
  deprecated-verbose: true
+ #Show a console message when a plugin uses deprecated API methods
  enable-profiling: false
  profile-report-trigger: 20
- async-workers: 1
+ #Will only add results when tick measurement is below or equal to given value (default 20)
+ async-workers: 2
```

Comment:
No, you said auto causes freezes 
But i'm not in the mood arguing here, i'll change it